### DISCUSSION TITLE
Why are several models I am loading answering prompts with only additional questions?

### DISCUSSION THREAD

**Discussion by curvedinf** [oobabooga/text-generation-webui#5219] [Q&A]
Several models I have loaded respond in this fashion:
```
AI
How can I help you today?
You
Write all needed models for a twitter clone
AI
Okay! What is your twitter clone's name?
You
Tweeter
AI
Is it an online platform where users can create profiles, follow other people and share posts with them, or something else?
```

It seems like my instruction template isn't working properly, but I did load the alpaca one as described in the help text and set the mode to chat-instruct. Does anyone know what's going on?

Settings:
![image](https://github.com/oobabooga/text-generation-webui/assets/40668638/cfd09859-b7f3-45a3-963d-5374b0b1c5b3)
![image](https://github.com/oobabooga/text-generation-webui/assets/40668638/30e79700-84a9-436c-a10b-4283f0d0f616)
![image](https://github.com/oobabooga/text-generation-webui/assets/40668638/d75d1709-d2a6-4e81-9d4a-0d24c7a1c9db)

**Comment by TechScorpion** [comment]
It probably won't write all of it but you said "Write all needed models for a twitter clone" - it might give more info if you wrote "Write all needed code for a twitter clone". But it also might need to ask you a few question first to make sure it gives you what you want.

I tried that with TheBloke_WizardCoder-Python-13B-V1.0-GPTQ and it gives a list of instructions and can give example code (see attached) - though doesn't always - it might depend on the prompt etc. I also increased the length of max_new_tokens to 4096 for the test with "divine intellect" preset and ExLlama_HF loader (not sure if a different loader might be better like ExLlama). In the instructions template tab it show Alpaca and I haven't changed anything there (I assume it's the default).
![write-code-for-twitter-clone-test](https://github.com/oobabooga/text-generation-webui/assets/146674825/d1857994-4874-49a4-8263-efb81fb371d7)
